tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -712,4 +712,4 @@ class AdaptiveQueryExecSuite
     spark.read.parquet(path).createOrReplaceTempView(name)
   }
 
-}
\ No newline at end of file
+}
```

Comment:
nit could we revert the white space only change to this file?

Reply:
Fixed.